base/linalg/lapack.jl

```diff
@@ -664,7 +675,7 @@ for (gesvx, elty) in
             ferr      = similar(A, $elty, nrhs)
             berr      = similar(A, $elty, nrhs)
             work      = Array($elty, 4n)
-            iwork     = Array($elty, n)
+            iwork     = Array($elty, 2n)
```

Comment:
Trying with `2n` instead of `4n` causes the following intermittent segfault:

```
signal (11): Segmentation fault: 11
__pool_alloc at /Users/kshyatt/Projects/julia/src/gc.c:1060
_new_array_ at /Users/kshyatt/Projects/julia/src/array.c:84
_new_array at /Users/kshyatt/Projects/julia/src/array.c:379
jl_exprn at /Users/kshyatt/Projects/julia/src/alloc.c:838
jl_deserialize_value_ at /Users/kshyatt/Projects/julia/src/dump.c:1131
jl_deserialize_value_ at /Users/kshyatt/Projects/julia/src/dump.c:1138
jl_uncompress_ast at /Users/kshyatt/Projects/julia/src/dump.c:1715
jl_prepare_ast at /Users/kshyatt/Projects/julia/src/ast.c:923
copy_ast at /Users/kshyatt/Projects/julia/src/ast.c:768
jl_cellset at /Users/kshyatt/Projects/julia/src/./julia.h:658
jl_cellset at /Users/kshyatt/Projects/julia/src/./julia.h:658
jl_cellset at /Users/kshyatt/Projects/julia/src/./julia.h:658
jl_cellset at /Users/kshyatt/Projects/julia/src/./julia.h:658
jl_prepare_ast at /Users/kshyatt/Projects/julia/src/ast.c:924
typeinf_uncached at ./inference.jl:1410
jlcall_typeinf_uncached_35 at /Users/kshyatt/Projects/julia/usr/lib/julia/sys.dylib (unknown line)
typeinf at ./inference.jl:1326
jlcall_typeinf_4 at /Users/kshyatt/Projects/julia/usr/lib/julia/sys.dylib (unknown line)
typeinf_ext at ./inference.jl:1270
jl_apply at /Users/kshyatt/Projects/julia/src/./julia.h:1260
jl_mt_assoc_by_type at /Users/kshyatt/Projects/julia/src/gf.c:939
jl_apply_generic at /Users/kshyatt/Projects/julia/src/gf.c:1640
_start at ./client.jl:452
jlcall__start_18941 at /Users/kshyatt/Projects/julia/usr/lib/julia/sys.dylib (unknown line)
true_main at /Users/kshyatt/Projects/julia/./julia (unknown line)
main at /Users/kshyatt/Projects/julia/./julia (unknown line)
Segmentation fault: 11
```